style.css

```diff
@@ -617,6 +617,106 @@ textarea::placeholder {
   font-size: 20px;
 }
 
+/* --- Popup Section --- */
+.overlay {
+  width: 100vw;
+  height: 100vh;
+  background: rgba(0, 0, 0, 0.6);
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  display: none;
+  backdrop-filter: blur(10px);
+}
+
+.popup-contain {
+  width: 50%;
+  margin: 0 auto;
+  padding: 24px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  z-index: 2;
+  background-color: #fff;
+  box-sizing: border-box;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.justify-content-space-between {
+  justify-content: space-between;
+  margin: 0 12px;
+}
+
+.links {
+  margin: 1rem 0 2rem 0;
+}
+
+.links ul {
+  display: flex;
+}
+
+.popup-content {
+  display: grid;
+  grid-template-columns: 1.4fr 1fr;
+  grid-template-rows: 1fr;
+  gap: 1rem;
+}
+
+.pop-img {
+  width: 100%;
+  height: 100%;
+}
+
+.pop-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  width: 100%;
+  height: 82px;
+  margin-bottom: 12px;
+}
+
+.pop-txt {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 30px;
+}
+
```

Comment:
You did a great job with the mobile version of your popup modal, However, as suggested by the previous reviewer about your mobile popup, there is still a little adjustment to be made. Your popup screen doesn't have the full view of your project (that is, some part of the view is truncated)as it is in the Figma template you were given. kindly use the right properties so it looks close to the design as much as possible 😄 

Your design :
![image](https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/20537712/1c88d7c3-dfce-4e7b-ad93-d4266ab03778)

Figma design :
![image](https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/20537712/967a0fc1-44ce-4d91-9074-903dc47135f5)

Reply:
Kindly provide me a better screenshot the problem to understand it better. Thank you

Reply:
@usorfaitheloho Kindly provide me a better screenshot the problem to understand it better. Thank you

Reply:
@Mahdi-Niazi The screenshot I provided is how your design looks, kindly compare it with the Figma design. I think you should increase the height of your modal popup so it captures your full design 😄